app/controllers/users_controller.rb

```diff
@@ -22,9 +28,60 @@ def create
     end
   end
 
+  def update
+    if @user.update_attributes user_params
+      flash[:success] = t ".success"
+      redirect_to @user
+    else
+      flash.now[:warning] = t ".fails"
+      render :edit
+    end
+  end
+
+  def destroy
+    if @user.destroy
+      flash[:success] = t ".delete"
+      redirect_to users_url
+    else
+      flash.now[:danger] = t ".fail"
+      render :index
```

Comment:
`redirect_to users_url` với `render :index` khác nhau thế nào?
refactor this code